src/app/shared/shared.module.ts

```diff
@@ -21,18 +23,30 @@ const declarations = [components, directives, pipes].reduce(
     []
 );
 
+// services which will be loaded in different way
+const skipServices = ['AuthorizedHttp'];
 // Get only providers, not types or other objects
-const providers = Object.keys(services)
+let providers = Object.keys(services)
```

Comment:
to moze lepiej zostawic const providers i zamiast przypisania providers = provider.concat zrobic providers.push.